app/controllers/skills_controller.rb

```diff
@@ -23,11 +23,15 @@ def create
   end
 
   def edit
-
   end
 
   def update
     @skill.update(skill_params)
+    if  @skill.save
+      redirect_to @skill, notice: 'Skill was sucessfully edited'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.